MinimOsd_Extra/protocols/NMEA.h

```diff
@@ -316,14 +300,15 @@ store to msg.nmea.lat
 
 
 void read_NMEA(){
+    char string[NMEA_BUF_LENGTH];
     while (Serial.available_S()) {
```

Comment:
Thank you so much for your work and time. 
I try it for a 3 times, firmware update to OSD with eeprom reset but get only dark screen. Nothing! File: MinimOsd_Extra_Uni.937DV-NMEA-release.hex extracted from downloaded osd_latest.zip ... Sorry